plugins.json

```diff
@@ -1282,6 +1282,13 @@
         "date": "2022-08-23T18:35:02.945086+02:00",
         "sha512sum": "1620b341b3a58167fb8f6a8d7f731be6ada7b77e224d8c434b378a3b319612619cf42f959fdeb007c96169d20d45fa85d6e47414478c441edc04f27b19668d25",
         "requires": ">=22.04.0"
+      },
+      {
+        "version": "0.3.0",
+        "url": "https://github.com/nextflow-io/nf-quilt/releases/download/0.3.0/nf-quilt-0.3.0.zip",
```

Comment:
Think this is the wrong repo. Likely you need to update your build script with your own org

Reply:
Whoops, fixed manually for now. Will fix in the build script going forward.

Reply:
The initial meta was corrupt due to a missing `gradle.properties`. Pushed a new 0.3.1 that has the correct URL, and added notes to the Makefile to detect and warn about the problem.